css.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  localIdentName: '[path][local]__[hash:base64:5]',
```

Comment:
is this..causing snapshot updates in the other pr?

Reply:
Nope.

Reply:
it is lodash that is causing the diffs :-(